docs/streams/kafka_on_gdn/index.md

```diff
@@ -0,0 +1,16 @@
+---
+sidebar_position: 150
```

Comment:
This is not where we define the sidebar. Look in one of the other folders and copy a `_category_.json` file from it to define the sidebar position and name for the folder.